pages/api/bib/[id]/items.ts

```diff
@@ -9,8 +10,9 @@ import { fetchBib } from "../../../../src/server/api/bib"
  */
 async function handler(req: NextApiRequest, res: NextApiResponse) {
   const id = req.query.id as string
-  const { discoveryBibResult, status } = await fetchBib(id, req.query)
-  const items = discoveryBibResult?.items
+  const viewAllItems = req.query?.view_all_items === "true"
+
+  const { status, items } = await fetchItems(id, req.query, viewAllItems)
```

Comment:
Nitpick: It feels a bit unnecessary to get the `id` from `req.query.id` and pass it to `fetchItems` as an argument when the entire `req.query` object is passed as the second argument.

Reply:
Same for `view_all_items` being checked. Would it make sense to pass the entire object and let `fetchItems` process the object? that function could be its own small function in `/src/server/api/items` that lives alongside `fetchItems`.

Reply:
Very good suggestions. Just updated fetchItems to just take a bibQuery param with the option to pass in a batch size query param for testing.